website/src/Pages/GalleryPage/SubmissionDetailsPage.jsx

```diff
@@ -121,103 +130,184 @@ function SubmissionDetailsPage() {
         </Box>
         <Box sx={{ display: 'flex' }}>
-          <Typography sx={{ width: 200, height: 30 }}>
+          <p className="details-label">
             Email:
-          </Typography>
-          <Typography>
+          </p>
+          <p className="details-text">
             {details.email}
-          </Typography>
+          </p>
         </Box>
         <Box sx={{ display: 'flex' }}>
-          <Typography sx={{ width: 200, height: 30 }}>
+          <p className="details-label">
             Description:
-          </Typography>
+          </p>
           {(edit) ? (
             <TextField
               value={description}
               onChange={(e) => setDescription(e.target.value)}
+              inputProps={{
+                style: {
+                  color: '#53595C',
+                  border: '0.75px solid #8A9195',
+                  borderRadius: '5px',
+                  backgroundColor: '#F8F8F8',
+                },
+              }}
             />
           ) : (
-            <Typography>
+            <p className="details-text">
               {description}
-            </Typography>
+            </p>
           )}
         </Box>
         <Box sx={{ display: 'flex' }}>
-          <Typography sx={{ width: 200, height: 30 }}>
+          <p className="details-label">
             Media Type:
-          </Typography>
-          <Typography>
+          </p>
+          <div className="media-type-container">
             {details.mediaTypes.map((type) => (
-              <Typography>{type}</Typography>
+              <div className="media-type-item">
+                {mediaTypeIconMap[type]}
+                <p className="media-type-text">{type}</p>
+              </div>
             ))}
-          </Typography>
+          </div>
         </Box>
         <Box sx={{ display: 'flex' }}>
-          <Typography sx={{ width: 200, height: 30 }}>
+          <p className="details-label">
             Tags:
-          </Typography>
-          <Box sx={{ display: 'flex' }}>
+          </p>
+          <div className="tags-container">
             {details.tags.map((tag) => (
-              <Typography sx={{ marginRight: 1 }}>{tag}</Typography>
+              <p className="tag">{tag}</p>
             ))}
-          </Box>
+          </div>
         </Box>
-        <Box>
+        <Box sx={{ mt: 5 }}>
           {details.mediaData.map((media) => (
-          // TODO: adjust the rendering for each type of media
-            <img style={{ height: 300, width: 300 }} src={media.MediaURL} alt={media.ContentType} />
+            <div>
+              {media.ContentType?.startsWith('image/') && (
+              <img
+                style={{
+                  width: '100%', borderRadius: 5, marginBottom: 20, border: '1px solid #4C4C9B',
+                }}
+                src={media.MediaURL}
+                alt={media.ContentType}
+              />
+              )}
+              {media.ContentType?.startsWith('video/') && (
```

Comment:
Once you approve the gallery media render, you can decide if you want to go with your method or mine